pkg/common/cluster_actions.go

```diff
@@ -157,6 +159,14 @@ type Domain struct {
 	URL         string
 }
 
+/**
+* Call the endpoint configured in the client's APIDomian field. If the response can be parsed into into a list of domains,
```

Comment:
```suggestion
* Call the endpoint configured in the client's APIDomian field. If the response can be parsed into a list of domains,
```

Just putting this comment suggestion in for future changes, no need to go through CI again just for this suggestion.